test/integration/register-foreign-device.spec.ts

```diff
@@ -1,7 +1,7 @@
 import test from 'node:test'
 import assert from 'node:assert'
 
-import * as services from '../../src/lib/services/index'
+import { ServicesMap } from '../../src/lib/services'
```

Comment:
IMO importing service map makes sense only in client as we access services dinamically like

serviceMap[serviceName].

In all other places I would just import the explicit service/services that are needed and use them directly.